src/Connector/CachingConnector.php

```diff
@@ -76,6 +76,6 @@ public function isCacheEnabled()
 
     private function hash(array $structure)
     {
-        return json_encode($structure);
+        return preg_replace('/[\{\}\(\)\/\\\@:]/', '.', json_encode($structure));
```

Comment:
Was there any particular reason for choosing the dot (`.`) as a replacement character?